.circleci/tests/test_confounds.py

```diff
@@ -0,0 +1,72 @@
+"""Test confounds handling."""
+import os
+
+import numpy as np
+import pandas as pd
+from nilearn.glm.first_level import make_first_level_design_matrix
+
+from xcp_d.utils.confounds import load_confound_matrix
+
+
+def test_custom_confounds(data_dir, tmp_path_factory):
```

Comment:
We can use this basic approach for testing out the different confound strategies.